File: mo-common/mo-common-sms/src/main/java/ink/ckx/mo/common/sms/query/SmsCodeQuery.java
package ink.ckx.mo.common.sms.query;

import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotBlank;
import lombok.Data;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/14
 */
@Schema(description = "短信验证码Query")
@Data
public class SmsCodeQuery {

    @NotBlank(message = "手机号不能为空")
    @Schema(description = "手机号")
    private String phoneNumber;
}

File: mo-common/mo-common-sms/src/main/kotlin/ink/ckx/mo/common/sms/query/SmsCodeQuery.kt
package ink.ckx.mo.common.sms.query

import io.swagger.v3.oas.annotations.media.Schema
import jakarta.validation.constraints.NotBlank

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/14
 */
@Schema(description = "短信验证码Query")
data class SmsCodeQuery(

    @field:NotBlank(message = "手机号不能为空")
    @Schema(description = "手机号")
    var phoneNumber: String? = null,
)